nodejs/json-rpc-relay/integration.tests/consensus.nodes/eth.sendRawTransaction.test.js

```diff
@@ -0,0 +1,48 @@
+console.clear();
+
+const { ContractId } = require('@hashgraph/sdk');
+
+require('dotenv').config({path: '.env'});
+
+const { ethers } = require("hardhat");
+
+const operatorPrivateKey = process.env.OPERATOR_PRIVATE_KEY;
+const communityHashioTestnet = process.env.COMMUNITY_TESTNET_URL;
+const arkhiaJsonRpcRelayTestnet  = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+
+describe('RPC', () => {
+  const init_message = 'initial_msg';
+  let contractAddress;
+  let wallet;
+  let provider;
+  let greeter;
+  let contractID;
+
+  beforeAll(async () => { 
+    provider = new ethers.providers.JsonRpcProvider(arkhiaJsonRpcRelayTestnet);
+    wallet = new ethers.Wallet(operatorPrivateKey, provider);
+    const Greeter = await ethers.getContractFactory('Greeter', wallet);
+    greeter = await Greeter.deploy(init_message);
+    const receipt = await greeter.deployTransaction.wait();
+    console.log('receipt: ', receipt);
+    contractAddress = receipt.contractAddress;
+    contractID = ContractId.fromSolidityAddress(contractAddress);
+    console.log(contractID, contractAddress);
```

Comment:
i think we should remove these console.logs

its better to expand below on your `should be able to deploy a contract`. and assess more like the contractID